.vscode/launch.json

```diff
@@ -153,7 +153,17 @@
 			"cwd": "${workspaceFolder}/_examples/widget_demos/flipbook",
 			"env": {},
 			"args": []
-		},
+    },
```

Comment:
Hate to be a pain but can you fix the formatting - align the braces and data with the rest of the document?